R/learner_survivalsvm_surv_svm.R

```diff
@@ -18,14 +25,19 @@
 #'
 #' `makediff3` is recommended when using `type = "hybrid"`.
 #'
+#' @section Custom mlr3 parameters:
+#'
+#' - `gamma`, `mu` have replaced `gamma.mu` so that it's easier to tune these separately.
```

Comment:
just fyi: This could have easily been achieved with a parameter trafo as well

Reply:
We had some much trouble with this doing it with the trafo, Lukas had made an issue about it => https://github.com/slds-lmu/paper_2023_survival_benchmark/issues/9